Tests/AestraAudio/ArsenalRouteModeRoundTripTest.cpp

```
@@ -0,0 +1,151 @@
+    const UnitInfo* preview = um.getUnit(1);
+    const UnitInfo* track = um.getUnit(2);
+    require(preview != nullptr, "Legacy preview unit missing");
+    require(track != nullptr, "Legacy track unit missing");
+    require(preview->routeMode == ArsenalRouteMode::PreviewToMaster,
+            "Legacy preview unit routeMode should resolve from routeId");
+    require(track->routeMode == ArsenalRouteMode::RoutedToTimelineTrack,
+            "Legacy track unit routeMode should resolve from routeId");
+}
+} // namespace
+
+int main() {
```

Cppcheck (2.20.0)

[style] The function 'toString' is never used.

(unusedFunction)